queue_job/models/base.py

```diff
@@ -216,3 +216,29 @@ def _job_store_values(self, job):
         :return: dictionary for setting job values.
         """
         return {}
+
+    @api.model
+    def _job_prepare_context_before_enqueue_keys(self):
+        """Keys to keep in context of stored jobs
+
+        Empty by default for backward compatibility.
```

Comment:
I must think more about it but my first intuition is that we should indeed keep it empty and enable this list by default in 16.0.

Reply:
Might be a wise choice :)

Reply:
And it's easy to override the empty list in new projects or if we accept the risk.
Probably better to do it this way, yeah. I'll leave the code as-is to see if we can agree on the *target* for 16.0, then make the backward compatible version (which will be: empty the list, add more info in the readme, adapt the tests).